graphql/schema.graphql

```diff
@@ -16,6 +16,17 @@ type Query {
     loadout(id: ID @eq): Loadout @find
     guns: [Gun!]! @paginate(defaultCount: 10)
     gun(id: ID @eq): Gun @find
+    equipments: [Equipment!]! @paginate(defaultCount: 20)
```

Comment:
Hmm, this seems weird, but I am not sure what to do. You can't have the same query name and different arguments. I am wondering if there is a standard way to name this better.

Reply:
Yeah I was in the same boat. Obviously "equipments" is ridiculous, but it was the simplest thing to do at the time. If there is a better way, happy to adjust. Otherwise we might just have to refactor once we think of a smarter name.

Reply:
TODO: Resolve later